api/src/test/java/org/pmiops/workbench/initialcredits/InitialCreditsExpirationServiceTest.java

```diff
@@ -107,10 +110,12 @@ public void test_userBypassed() {
     assertThat(service.getCreditsExpiration(user)).isEmpty();
   }
 
-  // TODO RW-13502
-  //  @Test
-  //  public void test_institutionBypassed() {
-  //  }
+  @Test
+  public void test_institutionBypassed() {
+    DbUser user = spyUserDao.save(new DbUser());
+    when(institutionService.shouldBypassForCreditsExpiration(user)).thenReturn(true);
```

Comment:
Does this test still pass without this line?  I think it would, because the user doesn't have a `UserInitialCreditsExpiration`.  Let's add that, to show that institutional bypass is the specific reason we don't see an expiration time.

Reply:
It works! Good call! Thanks.

Reply:
Good point. I updated the wording:
"If true, this institution's users are exempt from credit expiration."

How does that sound?